.github/workflows/docker.yml

```diff
@@ -0,0 +1,45 @@
+name: docker
+
+on:
+  push:
+    branches:
+      - main
+  release:
+    types:
+      - published
+
+jobs:
+  containerize:
+    strategy:
+      matrix:
+        app:
+          - Api
+          # - AdminConsole
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
+      - name: Install Docker Buildx
+        uses: docker/setup-buildx-action@f95db51fddba0c2d1ec667646a06c2ce06100226 # 3.0.0
+
+      - name: Login to DockerHub
+        run: >
+          echo ${{ secrets.DOCKER_TOKEN }} |
+          docker login --username PLEASE_FILL_ME --password-stdin
+
+      - name: Build & push image
+        run: >
+          docker buildx build
+          --file ${{ matrix.app }}.dockerfile
+          --platform linux/amd64,linux/arm64
+          --push
+          --tag passwordless/api-test-server:latest
+          ${{ github.event_name == 'release' && '--tag passwordless/api-test-server:$GITHUB_REF_NAME' || '' }}
+          ${{ github.event_name == 'release' && '--tag passwordless/api-test-server:stable' || '' }}
+          .
```

Comment:
Using `buildx` here to build for multiple platforms at the same time and to pack them all in one image bundle.

Reply:
The tags are set accordingly:

`:latest` -- last `main` branch commit or release
`:stable` -- last release
`:1.2.3` -- specific release